substrate/frame/support/test/tests/runtime_ui/missing_runtime.stderr

```diff
@@ -0,0 +1,5 @@
+error: Missing `#[runtime::runtime]`
```

Comment:
It'd probably be more helpful if we add a hint like `Add a struct inside the module and annotate it with #[runtime::runtime]`. Right now it looks like you're missing that attribute on the mod itself.